start-site/src/main/resources/application.yml

```diff
@@ -1292,4 +1292,4 @@ spring:
     restart:
       additional_paths:
         - start-client/build/
-      additional_exclude: "**/*.js,**/*.css"
\ No newline at end of file
+      additional_exclude: "**/*.js,**/*.css"
```

Comment:
Can you please remove this line and push-force an update?

Reply:
done.